presto-record-decoder/src/main/java/io/prestosql/decoder/avro/AvroColumnDecoder.java

```diff
@@ -166,8 +166,8 @@ public boolean isNull()
         @Override
```

Comment:
Can we have a test that would expose this bug, so the then we could the fix that actually fixes the problem (test)?

Reply:
ping